File: src/main/java/com/eason/Student/Student.java
package com.eason.Student;

public class Student {
    String id;
    String name;
    int english;
    int math;

    public Student(String name, int english, int math) {
        this.name = name;
        this.english = english;
        this.math = math;
    }
    public void print(){
        System.out.println(name + "\n" + english + "\t"+ math +
                "\t" + "平均分數:"+(english + math)/2 );
    }
}

File: src/main/kotlin/com/eason/Student/Student.kt
package com.eason.Student

import java.util.*

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    print("Please enter Student's name: ")
    var name = scanner.next()
    print("Please enter Student's english: ")
    var english = scanner.nextInt()
    print("Please enter Student's math: ")
    var math = scanner.nextInt()

    val stu = Student(name , english, math)
    stu.print()
}


class Student(var name:String, var english:Int, var math: Int) {
    fun print(){
        println(name +"\n"+ english + "\t" + math + "\t"+ "平均分數: "+(english+math)/2)

    }

}